package/yast2.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Feb 26 10:42:42 CET 2020 - schubi@suse.de
+
+- Updated docu for SysctlConfig class (bsc#1151649).
```

Comment:
It is only about documentation, that is why I commented it could be done next time the class was touched or the package, thus, I think that we can omit the version bump

Reply:
https://github.com/yast/yast-network/pull/1041#pullrequestreview-364702028